_posts/simulation-machine/individual-claims-history-simulation-machine.Rmd

```diff
@@ -0,0 +1,134 @@
+---
+title: "simulationmachine: Synthesizing Individual Claims Data"
+description: |
+  Generating claims data that can be used for loss reserving research and backtesting.
+authors:
+  - name: "Andrea Gabrielli"
+    url: https://math.ethz.ch/utils/search.html?persid=173657&pagetype=people
+    affiliation: RiskLab, ETH Zürich
+    affiliation_url: https://risklab.ch/
+  - name: "Kevin Kuo"
+    url: https://kevinykuo.com/
+    affiliation: Kasa AI
+    affiliation_url: https://kasa.ai/
+  - name: "Mario V. Wüthrich"
+    url: https://people.math.ethz.ch/~wueth/
+    affiliation: RiskLab, ETH Zürich
+    affiliation_url: https://risklab.ch/
+slug: simulationmachine
+bibliography: bibliography.bib
+output: distill::distill_article
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+Today we're excited to announce the [simulationmachine](https://github.com/kasaai/simulationmachine) package, based on the paper *An Individual Claims History Simulation Machine*[@gabrielli2018individual]. We created this package for the practitioner or researcher who needs realistic data for testing reserving or claims analytics modeling workflows.
+
+The package allows you to simulate individual claim characteristics along with their cash flow histories. We first show a quick example before delving into details of the algorithm.
+
+## simulationmachine
+
+You can download simulationmachine from GitHub with
+
+```r
+# install.packages("remotes")
+remotes::install_github("kasaai/simulationmachine")
+```
+
+First, specify the desired characteristics of the simulation:
+
+```{r echo=TRUE}
+library(simulationmachine)
+
+charm <- simulation_machine(
+  num_claims = 50000, 
+  lob_distribution = c(0.25, 0.25, 0.30, 0.20), 
+  inflation = c(0.01, 0.01, 0.01, 0.01), 
+  sd_claim = 0.85, 
+  sd_recovery = 0.75
+)
+charm
+```
+
+Here, we are noting that we would like 
+
+- Approximately 50,000 claims (note that this is an expected value, as claim incidence is also simulated)
+- The distribution among the four lines of business is 25%, 25%, 30%, and 20%
+- Inflation is a constant 1% for all lines of business
+- Claims are lognormally distributed with a standard deivation of 0.85
+- Recoveries are lognormally distributed with a standard deviation of 0.75
+
+Once we have the specification, we can perform the actual simulation using the `conjure()` function:
+
+```{r, echo = TRUE, message = FALSE}
+library(tidyverse)
+records <- conjure(charm, seed = 100)
+glimpse(records)
```

Comment:
I stared at 600k for a min before I guessed it was 50k claims x 12 periods of obs. I know this is highlighted at the bottom, but I think more context on the details from the paper would be helpful. Before I got to the end of the paper, I didn't realize that the `simulationmachine` was tied to a specific dataset. This makes the parameters and some of the outputs (e.g., lognormal distribution and AY 1994 - 2005) make more sense. Perhaps, adding a brief summary of the paper (a condensed version of 2nd paragraph in 'Motivations & Technical Details' section) to the end of the introduction section would help to "set the stage" for readers.

Separately from that, I think a brief comment of why 600k records here is worthwhile.